tests/BaseTest.php

```diff
@@ -178,7 +178,8 @@ protected function getAuditManager(): AuditManager
         }
 
         $auditConfig = AuditConfiguration::forEntities($this->auditedEntities);
-        $auditConfig->setGlobalIgnoreColumns(['ignoreme']);
+        $auditConfig->setGlobalIgnoreColumns(['ignoreMe']);
+        $auditConfig->setEntityIgnoredProperties(['SimpleThings\EntityAudit\Tests\Fixtures\Core\ProfileAudit' => ['ignoreProperty']]);
```

Comment:
Could we use the `class` pseudo-constant for the `\SimpleThings\EntityAudit\Tests\Fixtures\Core\ProfileAudit` class?